openff/interchange/_tests/__init__.py

```diff
@@ -76,9 +76,9 @@ class MoleculeWithConformer(Molecule):
     """Thin wrapper around `Molecule` to produce an instance with a conformer in one call."""
 
     @classmethod
-    def from_smiles(self, smiles, name="", **kwargs):
+    def from_smiles(self, smiles, name="", *args, **kwargs):
         """Create from smiles and generate a single conformer."""
-        molecule = super().from_smiles(smiles, **kwargs)
+        molecule = super().from_smiles(smiles, *args, **kwargs)
```

Comment:
(not blocking) seems to be irrelevant to this changeset (not used anywhere else in the PR) but it makes sense as a standalone change.

Reply:
Right, I think there was a reason in some tinkering that didn't make it to the end. I agree it stands on its own so I left it in